README.md

```diff
@@ -53,8 +53,25 @@ the open source community.
 
 ## Quick Start Guide
 
-The easiest way to deploy Volcano is to use the Helm chart.
+The easiest way to deploy Volcano is to use the Helm chart.  Volcano can be installed by cloning new code and also without cloning code.
 
+## Without Cloning Code
+### Helm Charts
+
+Add Helm repo using following command,
+
+```
+helm repo add volcano https://raw.githubusercontent.com/volcano-sh/volcano/master/installer/helm_package
```

Comment:
The helm package sounds more like a release collection of volcano charts. therefore I have some questions here :)
1. why we put the release versions here in code?
2. it's possible that the helm package will not reflect the latest codes, should we add some notes here?
3. who & when to sync the tgz when releasing new version?

Reply:
1. We can have multiple versions or releases of helm charts, few projects maintain that, that might not be required for us, but it is mandatory to provide version for helm,
2. Yes, if any changes made in helm charts, should be reflected back to the package by the developer, who makes those changes, have included a simple README in installer/helm_package folder, they can follow that and update the chart package,
3. Developer who make changes in charts should update tgz file when a change is made in installer/chart/volcano folder.